frontend/kesaseteli/employer/src/components/application/summary/EmploymentSummary.tsx

```diff
@@ -0,0 +1,121 @@
+import EmploymentFieldSummary from 'kesaseteli/employer/components/application/summary/EmploymentFieldSummary';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import FormSectionHeading from 'shared/components/forms/section/FormSectionHeading';
+import Employment from 'shared/types/employment';
+import { convertToUIDateFormat } from 'shared/utils/date.utils';
+
+type Props = {
+  employment: Employment;
+  index: number;
+};
+
+const EmploymentSummary: React.FC<Props> = ({ index, employment }) => {
+  const {
+    id,
+    summer_voucher_exception_reason,
+    employee_name,
+    employee_ssn,
+    employment_start_date, // yyyy-MM-dd
+    employment_end_date, // yyyy-MM-dd
+    employment_work_hours,
+    employment_salary_paid,
+    employment_description,
+    hired_without_voucher_assessment,
+  } = employment;
+
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <FormSectionHeading
+        header={`${employee_name ?? ''} ${employee_ssn ?? ''}`}
```

Comment:
Nitpicking, just in case `employee_ssn` is defined while `employee_name` is not 😉 
```suggestion
        header={[employee_name ?? '', employee_ssn ?? ''].join(' ').trim()}
```
Feel free to ignore.

Reply:
These values are never actually empty on summary page as they are required inputs in the form. So no need to do anything imho.